Mathlib/Data/Complex/Basic.lean

```diff
@@ -192,6 +207,8 @@ theorem add_im (z w : ℂ) : (z + w).im = z.im + w.im :=
   rfl
 #align complex.add_im Complex.add_im
 
+section
+set_option linter.deprecated false
```

Comment:
This section should only be around the `bit` lemmas, not the rest of the file.

Reply:
Thanks. Fixed (I thought the part calling the bit lemmas had to be in it too)